games/pbo/full/config.json

```diff
@@ -0,0 +1,151 @@
+{
+  "name": "Portraits",
+  "description": "Character portraits",
+  "prefix": "full_",
+  "postfix": "_",
+  "type": [
+    "full"
+  ],
+  "credits": "Files ripped from PBO Press Kit \n@Bloo4LYFE for cropping",
+  "version": "1.01",
+  "eyesights": {
```

Comment:
I found two artworks where the eyesight seems to be offset:
- Cocoa
![full_Cocoa_0](https://user-images.githubusercontent.com/1964201/231428446-c6c48ed2-468c-4fe3-941d-439ce9aac658.png)
- Rin and Gaito (Note: This one is debatable because of the two-character dynamic)
![full_RinGaito_0](https://user-images.githubusercontent.com/1964201/231428451-03a1034e-e918-406f-9be9-994a83b850fa.png)